gulpfile.js

```diff
@@ -132,7 +154,7 @@ gulp.task("test", (done) => {
     });
 });
 
-gulp.task("build", gulp.series("clean", "ts-compile", "html-webpack", "node_modules-webpack"));
+gulp.task("build", gulp.series("clean", "ts-compile", "html-webpack", "node_modules-webpack", /* "insert-serial-monitor-cli" */));

```

Comment:
```suggestion
gulp.task("build", gulp.series("clean", "ts-compile", "html-webpack", "node_modules-webpack", "insert-serial-monitor-cli"));

```

This should be good to go, there is a new release of serial-monitor-cli.